spec/features/print_spec.rb

```diff
@@ -0,0 +1,11 @@
+# typed: false
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Print terms of deposit' do
```

Comment:
Can you make this a request spec rather than a feature spec?  The request spec is much faster as it doesn't require a browser to be spun up.

Reply:
Done